docs/src/intro.rst

```diff
@@ -9,14 +9,10 @@ topics from documents, as efficiently (computer-wise) and painlessly (human-wise
 
 
 Gensim is designed to process raw, unstructured digital texts ("*plain text*").
-The algorithms in `gensim`, such as **Latent Semantic Analysis**, **Latent Dirichlet Allocation** and **Random Projections**
-discover semantic structure of documents by examining statistical
-co-occurrence patterns of the words within a corpus of training documents.
-These algorithms are unsupervised, which means no human input is necessary -- you only need a corpus of plain text documents.
 
-Once these statistical patterns are found, any plain text documents can be succinctly
-expressed in the new, semantic representation and queried for topical similarity
-against other documents.
+The algorithms in Gensim, such as **Word2Vec**, **FastText**, **Latent Semantic Analysis**, **Latent Dirichlet Allocation** and **Random Projections**, discover semantic structure of documents by examining statistical co-occurrence patterns within a corpus of training documents. These algorithms are **unsupervised**, which means no human input is necessary -- you only need a corpus of plain text documents.
```

Comment:
please add links to documentation pages for each model, btw, who use RandomProjections? Maybe better to add something like TfIdfModel or Doc2Vec (but not RandomProjections)

Reply:
OK.